package.json

```diff
@@ -6,7 +6,7 @@
     "node": "18 || 20"
   },
   "scripts": {
-    "dev": "concurrently \"yarn start --config ../../app-config.local.yaml\" \"NODE_OPTIONS=--no-node-snapshot yarn start-backend --config ../../app-config.local.yaml\"",
+    "dev": "concurrently \"yarn start --config ../../app-config.local.yaml\" \"NODE_OPTIONS=--no-node-snapshot yarn start-backend --inspect --config ../../app-config.local.yaml\"",
```

Comment:
That should be passed in another PR and ideally we should have (to be confirmed) 2 scripts; one to launch and another to debug

Reply:
ok, I will remove it from here